index.d.ts

```diff
@@ -14,11 +14,9 @@
  * limitations under the License.
  */
 
-export * from './lib/api';
-export const devices: typeof import('./lib/deviceDescriptors').DeviceDescriptors;
-export const errors: { TimeoutError: typeof import('./lib/errors').TimeoutError };
-export const chromium: import('./lib/server/chromium').Chromium;
-export const firefox: import('./lib/server/firefox').Firefox;
-export const webkit: import('./lib/server/webkit').WebKit;
-export const selectors: import('./lib/api').Selectors;
```

Comment:
What about selectors, web, errors and devices?

Reply:
Is web public right now? How is it accessed? I did the others.